.github/workflows/ecfr-parser-checks.yml

```diff
@@ -0,0 +1,57 @@
+name: eCFR Parser Checks
+
+on:
+  pull_request:
```

Comment:
Can we add 

  push:
    branches:
      - main

to this so that we are always running it on merge to main.  I know that seems redundant, but I think every merge to main should run these tests, especially if we are not running them on opening every PR.